cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -169,10 +169,10 @@ else()
   set(BROTLI_SOURCE_URL "https://github.com/google/brotli/archive/${BROTLI_VERSION}.tar.gz")
 endif()
 
-if (DEFINED ENV{DOUBLE_CONVERSION_SOURCE_URL})
-  set(DOUBLE_CONVERSION_SOURCE_URL "$ENV{DOUBLE_CONVERSION_SOURCE_URL}")
+if (DEFINED ENV{ARROW_DOUBLE_CONVERSION_URL})
```

Comment:
Renamed to follow `ARROW_{dependency}` naming convention.

Reply:
Hmm, right, but please keep `DOUBLE_CONVERSION_SOURCE_URL` for the internal CMake variable (as in `BROTLI_SOURCE_URL` above).

Reply:
Oops, fixed it!